src/core/control/settings/Settings.cpp

```diff
@@ -1812,18 +1866,8 @@ auto Settings::getButtonConfig(int id) -> ButtonConfig* {
     return this->buttonConfig[id];
 }
 
-auto Settings::getFullscreenHideElements() const -> string const& { return this->fullscreenHideElements; }
-
-void Settings::setFullscreenHideElements(string elements) {
-    this->fullscreenHideElements = std::move(elements);
-    save();
-}
-
-auto Settings::getPresentationHideElements() const -> string const& { return this->presentationHideElements; }
-
-void Settings::setPresentationHideElements(string elements) {
-    this->presentationHideElements = std::move(elements);
-    save();
+void Settings::setViewModeAttributes(size_t mode, std::string attributes) {
```

Comment:
Use `enum ViewMode` instaed of `size_t`

Reply:
Same as first [change-request](https://github.com/xournalpp/xournalpp/pull/4514#discussion_r1057764129)